orchestrate/obykeyset/cachedpipelinemap.go

```diff
@@ -8,23 +8,29 @@ import (
 	"github.com/relex/slog-agent/base/bsupport"
 	"github.com/relex/slog-agent/defs"
 	"github.com/relex/slog-agent/util"
+	"github.com/relex/slog-agent/util/localcachedmap"
 )
 
-// PipelineChannel is an alias to write-only channel of log records
-type PipelineChannel = chan<- []*base.LogRecord
+type globalPipelineChannelMap = localcachedmap.GlobalCachedMap[chan<- []*base.LogRecord, *pipelineChannelLocalBuffer]
 
-// PipelineChannelLocalBuffer acts as goroutine/worker-local cache and buffer for writing to PipelineChannel
-// It collects pending logs and only sends to channel when certain limit is reached
-type PipelineChannelLocalBuffer struct {
+type localPipelineChannelMap = localcachedmap.LocalCachedMap[chan<- []*base.LogRecord, *pipelineChannelLocalBuffer]
+
+func closePipelineChannel(ch chan<- []*base.LogRecord) {
```

Comment:
do we need a function for 1 line? should we inline this code ?

Reply:
inlined